### DISCUSSION TITLE
CMI Placeholder functionality for offline players

### DISCUSSION THREAD

**Discussion by Dracnoian** [Zrips/CMI#6345] [Enhancement]
Hey there.

I use quite a few CMI placeholders, and often in some rather...unorthodox and creative ways. I have honestly really, really enjoyed using CMI for my server, because the plugin is so extremely flexible, it can do almost anything. However, for a lot of the applications I'd like to use Placeholders in, I can't, because they don't show data for offline players. I'm using PlaceholderAPI's Parseother with them.

Is it possible for that functionality to be added to the CMI Placeholders?

**Comment by CloudeLecaw** [comment]
Would be nice but for now like u allready seens the answer is.. not possible. Specialy when its information attached to a player like meta. We used an external Plugin ro store information on the players in a database so the can be used by placeholders even if the Player is offline. Papi solutions like parseothers dont help here for cmis asked things over placeholders
